packages/core/src/shader/ShaderSystem.ts

```diff
@@ -66,7 +66,7 @@ export class ShaderSystem extends System
      *
      * @private
      */
-    private systemCheck(): void
+    systemCheck(): void
```

Comment:
Everything looks fine - my only comment is here. Why would a function be docced as private, yet not be private in typescript?

Reply:
There’s not really a concept of internal in TypeScript but if there was, this would be it.  We’ve made things public even though they are only referenced externally (eg _tingRGB). This function is overridden by the unsafe-eval package so to keep from throwing a linting error it’s easier to make it public even if it never shows up in the docs. I think there’s a way we can exclude stuff like this from typings when we get there.